Documentation/api-reference.md

```diff
@@ -128,6 +132,69 @@ Edge.sendEvent(experienceEvent: experienceEvent) { (handles: [EdgeEventHandle])
 }
 ```
 
+##### Example with DatastreamIdOverride
```

Comment:
nit: shall we use sentence like header: Example with Datastream ID override / Example with Datastream config override